pkg/controllers/pluginconfig/remote_cluster_test.go

```diff
@@ -210,6 +210,27 @@ var _ = Describe("Validate pluginConfig clusterName", Ordered, func() {
 			return releases
 		}).Should(ContainElement(gstruct.PointTo(gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{"Name": Equal("test-plugin")}))), "the helm release should be deployed to the remote cluster")
 
+		By("updating the pluginConfig")
+		_, err = clientutil.CreateOrPatch(test.Ctx, test.K8sClient, testPluginConfig, func() error {
+			// this value enables the template of another pod
+			testPluginConfig.Spec.OptionValues = append(testPluginConfig.Spec.OptionValues, greenhousev1alpha1.PluginOptionValue{Name: "enabled", Value: test.MustReturnJSONFor("true")})
+			return nil
+		})
+		Expect(err).ShouldNot(HaveOccurred(), "there should be no error updating the pluginConfig")
+		By("checkind the resources deployed to the remote cluster")
+		remoteK8sClient, err := clientutil.NewK8sClientFromRestClientGetter(remoteRestClientGetter)
```

Comment:
You can get the `remoteK8sClient` from the `test.StartControlPlane(...)` func in the `bootstrapRemoteCluster` func at the end of this file....